ajax/libs/slim.js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "slim.js",
+  "filename": "Slim.min.js",
```

Comment:
Looks like I can't use uglifyjs to minify Slim.js, @eavichay can you please take a look?